Lib/Const.py

```diff
@@ -0,0 +1,27 @@
+
+SUCCESS = 0
+FAILURE = 1
+
+#
+# The following constants are bits for storing run_tests.py argument options.
+# If user can run the testsuite and specify -H (for generate html), then 
+# the valid_options passed to TestRunnerBase constructor should have
+# OPT_GENERATE_HTML bit set.
+#
+# for boolean option arguments
+#
+OPT_GENERATE_HTML  = 0x00000001
```

Comment:
We can review all the OPT_* const as we will not use them.

Reply:
yes I forgot to remove that file from the repo, I don't think it's needed any longer

Reply:
I use SUCCESS and FAILURE in TestRunnerBase.py and Util.py.

Reply:
I moved them there, did I not push these changes?